cog/core/downtime.py

```
@@ -0,0 +1,85 @@
+    channel: Messageable = bot.get_channel(channel_id)
+
+    if not channel:
+        raise ValueError(f"Cannot get channel (id={channel}).")
+
+    if not isinstance(channel, Messageable):
+        raise ValueError(
+            f"{channel.name} (id={channel_id}, type={type(channel)}) is not messageable."
+        )
+
+    messages = await channel.history(
+        limit=None, after=after, before=before, oldest_first=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)